HogeFuga/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="example.watarusuzuki.android_example_using_danger">
+
+    <application
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WataruSuzuki/Android-Example-Using-Danger/blob/7bc72839ceeb91362b29abe3d6e45fdb9be34890/HogeFuga/app/src/main/AndroidManifest.xml#L5"></span>App is not indexable by Google Search; consider adding at least one Activity with an ACTION-VIEW intent filter. See issue explanation for more details.</td>
    </tr>
  </tbody>
</table>